utils.py

```diff
@@ -0,0 +1,42 @@
+import pandas as pd
```

Comment:
Adding module docstrings is good practice 🙂 

Also importing `Path` from `pathlib` to ensure that the paths we use are independent of OS (simply using `data/survey_data.csv` as a string might fail for Windows users, for instance).

```suggestion
'''Utility functions to be used within the project'''

import pandas as pd
from pathlib import Path
```